LAYOUT_STYLE_GUIDE.md

```diff
@@ -39,8 +38,7 @@ The [lesson example](./templates/lesson-example.md) and [project example](./temp
 
 ### Lesson layout
 
-~~~markdown
-
+```markdown
```

Comment:
do you have a formatter set to format on save ?

Reply:
I see this happening more, I guess the formatter's defaults are backticks :) 
I will note it as something that possibly has to be mentioned in the contributors guide!

Thanks for everything you're doing lately @Eveieve ! It's appreciated

Reply:
@Eveieve yes it's something we do want to adhere to. 

I think it may also be important for people to realize that when working on a repo with others, you shouldn't use your own formatter unless you're sure it adhere to thee style guide of the repo/company if there's one.